### PR TITLE
Fix Voting php warning on blog category pages

### PR DESCRIPTION
**Pull request by PhilETaylor**
Pull Request for Issue #16203

Create a Blog item menu, create some article for the blog. Enable Show Voting in the blog menu item.

PHP Notice: Undefined property: JCategoryNode::$rating_count in ../plugins/content/vote/tmpl/rating.php on line 45

Replaced PR https://github.com/joomla/joomla-cms/pull/16204

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on b12781339c236f556a3a6522c58d98757c8b135f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16206">issues.joomla.org/tracker/joomla-cms/16206</a>.</sub>

File: plugins/content/vote/tmpl/rating.php [hunk 1]
```diff
@@ -20,7 +20,12 @@
  * @var   string   $path     Path to this file
  */
 
-$rating = (int) @$row->rating;
+if ($context=='com_content.categories')
```
Comment (Quy) [inline plugins/content/vote/tmpl/rating.php]: cs: add space before and after `==`.

Comment (PhilETaylor) [reply plugins/content/vote/tmpl/rating.php]: Who needs phpCodeSniffer when we have @Quy on the case :) :) 

fixed.

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 3edccf58c2cd7b0c95cf5f5c1dc4374a514a51e6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16206">issues.joomla.org/tracker/joomla-cms/16206</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 3edccf58c2cd7b0c95cf5f5c1dc4374a514a51e6<br /><br />Fine here.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16206">issues.joomla.org/tracker/joomla-cms/16206</a>.</sub>

**Comment by infograf768** [comment]
RTC. Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16206">issues.joomla.org/tracker/joomla-cms/16206</a>.</sub>

**Pull request merged**